docs/API/data/preprocessing/MeanNormalization.md

```diff
@@ -0,0 +1,4 @@
+# ADLStream.data.preprocessing.MeanNormalizationScaler
+
+::: ADLStream.data.preprocessing.MeanNormalizationScaler
+    handler: python
```

Comment:
Change the name of this file to `MeanNormalizationScaler.md` (as it is in the mkdocs.yml file)